src/plugins/intel_cpu/src/dnnl_postops_composer.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Convinience wrapper class for handling OneDNN attributes & post ops.
+ * @file dnnl_postops_composer.h
+ */
+#pragma once
+
+#include <dnnl_types.h>
+
+#include <string>
+
+#include "cpu_memory.h"
+#include "memory_desc/cpu_memory_desc.h"
+#include "memory_desc/dnnl_blocked_memory_desc.h"
+#include "onednn/dnnl.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class Node;
+
+// so far the API only support per-Tensor or per-OC
+class DnnlPostOpsComposer {
+public:
+    DnnlPostOpsComposer(ov::intel_cpu::Node* node,
```

Comment:
Seems like cpu node is used only to get dnnl_engine. Maybe it is better to pass the engine directly?

Reply:
Good idea, changed!